Server/src/models/AuthenticationModel.ts

```diff
@@ -127,4 +127,33 @@ export class AuthenticationModel {
 
         let connection = getConnection();
     }
+
+
+    //for issue 146
```

Comment:
Do we need this comment?

Reply:
removed all unnecessary comments on the recent commit